src/js/helpers/pageAndSortHelper.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * pageAndSortHelper.js
+ * created by Jonathan Hill 01/07/2020
+ */
+
+export const pageAndSort = (data, sortFunction, page, limit, sortDirection, sortProperty) => {
```

Comment:
can you annotate what this returns?

Perhaps it should return the current page, the new total items if the limit has changed, etc?

Reply:
Yeah, I'll add comments.

A user will be passing the current page, and limit and the data, therefore the user will already know all of those facts.